### PR TITLE
Support Infiniband testing for tumbleweed

### PR DESCRIPTION
**Pull request by frankenmichl**
Add support for running InfiniBand tests on openSUSE Tumbleweed
Do some cleanups and add schedules for Tumbleweed.

Related ticket: [https://progress.opensuse.org/issues/104871](https://progress.opensuse.org/issues/104871)

Verification run: http://baremetal-support.qa.suse.de/tests/2194 and http://baremetal-support.qa.suse.de/tests/2195

### CODE REVIEW

File: schedule/kernel/ibtest-master-tumbleweed.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name:           ibtest-master-tumbleweed
+description:    >
+    The master node for the infiniband testsuite (hpc-testing)
+vars:
+    DESKTOP: textmode
+    DEVEL_TOOLS_REPO: https://download.opensuse.org/repositories/devel:/tools/openSUSE_Tumbleweed/devel:tools.repo
+    GA_REPO: http://dist.suse.de/ibs/SUSE:/SLE-%VERSION%:/GA/standard/SUSE:SLE-%VERSION%:GA.repo
```
Comment (b10n1k) [inline schedule/kernel/ibtest-master-tumbleweed.yaml:7]: is this used in TW?

File: tests/kernel/ibtests.pm [hunk 2]
```diff
@@ -107,22 +102,16 @@ sub run {
     $slave = get_required_var('IBTEST_IP2');
 
     $self->select_serial_terminal;
-
-    # unload firewall. MPI- and libfabric-tests require too many open ports
-    systemctl("stop " . opensusebasetest::firewall);
-
+    #
```
Comment (b10n1k) [inline tests/kernel/ibtests.pm:105]: leftover?

File: tests/kernel/ibtests_prepare.pm [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+use version_utils;
+use mmapi;
+
+
+sub permit_root_login {
+    my $self = shift;
+
+    select_console('sol', await_console => 1);
+    type_string "root\n";
+    sleep 1;
```
Comment (b10n1k) [inline tests/kernel/ibtests_prepare.pm:26]: i would rather use `wait_screen_change` in place of sleep. wdyt?

File: tests/kernel/ibtests_prepare.pm [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    type_string "\n";
+}
+
+sub run {
+    my $self = shift;
+    my $master = get_required_var('IBTEST_IP1');
+    my $slave = get_required_var('IBTEST_IP2');
+
+    my $role = get_required_var('IBTEST_ROLE');
+    my $packages = "rdma-core rdma-ndd iputils python";
```
Comment (b10n1k) [inline tests/kernel/ibtests_prepare.pm:42]: in my TW  python points to python2. 
```
❯ ll /usr/bin/python
lrwxrwxrwx 1 root root 9 Jun 15 04:10 /usr/bin/python -> python2.7
```
i wonder if this is what we want.

File: schedule/kernel/ibtest-master-tumbleweed.yaml (hunk 1 above)
Comment (frankenmichl) [reply schedule/kernel/ibtest-master-tumbleweed.yaml]: This should not be there, removed it.

**Review by b10n1k — APPROVED** [review]
+1

**Pull request merged**